ext/libxml/libxml_arginfo.h

```diff
@@ -42,7 +42,7 @@ static const zend_function_entry ext_functions[] = {
 	ZEND_FE(libxml_get_last_error, arginfo_libxml_get_last_error)
 	ZEND_FE(libxml_get_errors, arginfo_libxml_get_errors)
 	ZEND_FE(libxml_clear_errors, arginfo_libxml_clear_errors)
-	ZEND_FE(libxml_disable_entity_loader, arginfo_libxml_disable_entity_loader)
+	ZEND_DEP_FE(libxml_disable_entity_loader, arginfo_libxml_disable_entity_loader)
```

Comment:
This is a generated file, so you should add a `@deprecated` PHPDoc attribute to the function in the corresponding stub file instead.

Reply:
Thanks for pointing this out. I updated the PR just now.

Reply:
Looks good!